src/app/guide/_components/GuideDetail.tsx

```diff
@@ -0,0 +1,55 @@
+import Image from 'next/image';
+import SVGTriangle from '@/styles/icons/triangle.svg';
+
+interface ImageWithDescriptionProps {
+  src: string;
+  description: string;
+}
+function ImageWithDescription({ src, description }: ImageWithDescriptionProps) {
+  return (
+    <div className="flex w-full flex-col items-center">
+      <div className="relative w-full" style={{ paddingTop: '100%' }}>
+        <Image src={src} alt={description} layout="fill" objectFit="cover" />
+      </div>
+
+      <div className="font-primary-darkblue flex flex-row items-center gap-1 pt-2 font-sfpro text-xs font-extrabold">
+        <SVGTriangle />
+        {description}
+      </div>
+    </div>
+  );
+}
+
+interface GuideDetailProps {
+  title: string;
+  children: React.ReactNode;
+  examples: { id: string; src: string; description: string }[];
+}
+
+export default function GuideDetail({
+  title,
+  children,
+  examples,
+}: GuideDetailProps) {
+  return (
+    <div className="flex flex-col">
+      <div className="flex flex-row items-center pt-8">
+        <Image src="/profile.png" alt="Sample Image" width={72} height={72} />
+        <div className="pl-4">
+          <div className="text-xl">{title}</div>
+          <div className="font-sfpro text-sm text-white">{children}</div>
+        </div>
+      </div>
+
+      <div className="grid grid-cols-2 justify-center gap-x-4 gap-y-4 pt-4">
+        {examples.map((example) => (
+          <ImageWithDescription
+            key={example.id}
```

Comment:
```suggestion
            key={`guide-detail-${example.id}`}
```

같은 아이디를 공유하는 데이터를 가지는 컴포넌트가 페이지에 존재할 경우 오류가 발생할 수 있어요! 리액트 앱에서 항상 key는 unique 값을 가지도록 설정해야해요! (리액트 내부의 동작 방식에서 key값으로 컴포넌트와 요소를 구분하기 때문이에요.)